server/src/main/java/org/elasticsearch/action/support/broadcast/unpromotable/TransportBroadcastUnpromotableAction.java

```diff
@@ -64,6 +64,8 @@ protected TransportBroadcastUnpromotableAction(
         transportService.registerRequestHandler(
             transportUnpromotableAction,
             this.executor,
+            false,
+            true,
```

Comment:
Should this be false according to the PR description?

Reply:
bah thanks well spotted